cli/main.rs

```diff
@@ -750,6 +750,10 @@ async fn test_command(
       .filter(|e| !test_scripts.contains(&e.url))
       .filter(|e| !e.url.contains(".deno"))
       .filter(|e| !e.url.contains("__anonymous__"))
+      // XXX; just blackboxing these while I figure out why the source map doesn't resolve
+      .filter(|e| !e.url.contains("core.js"))
+      .filter(|e| !e.url.contains("rt/"))
+      .filter(|e| !e.url.contains("op_crates/"))
```

Comment:
Those file don't have sourcemaps - they are pure JS files :>

Reply:
Was filtering so that my test inputs was the only one failing with `.unwrap()`; those lines aren't staying ;)